content/docs/user-guide/gems/reference/aws/aws-gamelift/gem-setup.md

```diff
@@ -86,13 +86,14 @@ For a client application, your game must implement the following use cases to ma
 - `JoinSession`
 - `LeaveSession`
 
-The AWS GameLift Gem provides both [C++ APIs](cpp-api/) and [Scripting](scripting/) to support developers. You can implement these use cases using either method. 
+The AWS GameLift Gem provides both [C++ APIs](cpp-api/) and [scripting](scripting/) to support developers. You can implement these use cases using either method.  
```

Comment:
```suggestion
The AWS GameLift Gem provides both [C++ APIs](cpp-api/) and [scripting](scripting/). You can implement these use cases using either method.
```
Do we need to say "to support developers" here? Is that redundant, since the reader is already a game dev audience?